src/status_im/contexts/communities/events.cljs

```diff
@@ -248,3 +250,14 @@
                                  {:error   err
                                   :chat-id chat-id
                                   :event   "share-community-channel-url-with-data"}))}]})))
+(rf/reg-event-fx :communities/set-addresses-for-permissions
+ (fn [{:keys [db]} [addresses]]
+   {:db (assoc-in db [:communities/addresses-for-permissions] addresses)}))
+
+(rf/reg-event-fx :communities/update-last-opened-at
+ (fn [_ [community-id]]
+   {:json-rpc/call [{:method     "wakuext_communityUpdateLastOpenedAt"
+                     :params     [community-id]
+                     :on-success #(rf/dispatch [:communities/handle-community (first (:communities %))])
```

Comment:
what we are handling here? Do we need this?

Reply:
After returning the response of the community with the update LastOpenedAt value, We need to update the community in app-db for the changes to happen in the UI.

Reply:
Please can you elaborate, what changes will be update in UI after updating LastOpenedAt?

Reply:
The sorting of the communities, That'll be done in https://github.com/status-im/status-mobile/issues/17337

Reply:
@ibrkhalil, what we're doing here is very inefficient. Every time the community overview screen is opened we:

1. Update the timestamp on the backend.
2. Recursively transform a gigantic `Community` JS payload from scratch to a Clojure data structure.
3. If the community has token permissions (which many will), we re-dispatch other events, such as `:communities/check-permissions-to-join-community`, which may cause yet more db updates.

But why would we want to reprocess community permissions if the originating request just wanted to update a single timestamp column in the DB?

If we want to prioritize performance, we would make the event you wrote in this PR `:communities/update-last-opened-at` do a quick `assoc-in` in the app db and that's it. Much much cheaper.

Something along these lines:

```clojure
(rf/reg-event-fx :communities/update-last-opened-at
 (fn [_ [community-id]]
   {:json-rpc/call [{:method     "wakuext_communityUpdateLastOpenedAt"
                     :params     [community-id]
                     :on-success [:communities/update-last-opened-at-success]
                     :on-error   ...}]}))

(rf/reg-event-fx :communities/update-last-opened-at-success
 (fn [{:keys [db]} [^js community]]
   (let [community-id   (oops/oget community :id)
         last-opened-at (oops/oget community :lastOpenedAt)]
     {:db (assoc-in db [:communities community-id :last-opened-at] last-opened-at)})))
```

Reply:
What might be even better, If we could return just the value of the lastOpenedAt instead of returning the whole MessengerResponse, But I'd have to investigate if that's possible.

Reply:
Yes, that'd be even better 👍🏼 Thinking about the future, I would prefer if our clients could always request what data they need, similar to what's possible with GraphQL endpoints.

Reply:
Done